versions.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 0.12.9"
+  required_version = ">= 0.15.3"
```

Comment:
just setting this to what we used for the vpc module

Reply:
More importantly, are we looping these all into a new S3 backend? My guess is yes, but not sure if we should make a brand new one to avoid any potential contamination